components/ephemeral_storage/browser/ephemeral_storage_tab_helper.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_EPHEMERAL_STORAGE_BROWSER_EPHEMERAL_STORAGE_TAB_HELPER_H_
+#define BRAVE_COMPONENTS_EPHEMERAL_STORAGE_BROWSER_EPHEMERAL_STORAGE_TAB_HELPER_H_
+
+#include "content/public/browser/storage_partition_config.h"
```

Comment:
why is this here?

Reply:
Fixed. This was left over from a previous version of the change.